HW02_B_Mimic_starter.Rmd

```diff
@@ -52,7 +61,13 @@ data("iris")
 
 Using the iris dataset, make this graph:
 ```{r graph 2 code, echo=FALSE}
+ggplot(transform(iris,Species=factor(Species,levels=c("versicolor","setosa","virginica"))), aes(Sepal.Length, Petal.Length))+
+  geom_point(aes(shape=Species, color=Species))+
+  facet_wrap(Species, scales="free_y") +
+  geom_smooth(method="lm", se=F, color="black")
 
+#tried to put this in ggplot for instead of just "iris" to rearrange the Species, but it didn't work

```

Comment:
If you put this (~) in front of Species in line 66 of your code, I think that fixes your problem.